src/Client.php

```diff
@@ -439,6 +467,20 @@ public function send($method, $payload, $path, $params = [], array $headers = []
     }
 
     /**
+     * @param Request $request
+     * @param Response $response
+     * @param string $type
+     *
+     * @return File
+     */
+    protected function parsePdf(Request $request, Response $response, $type)
```

Comment:
commented on hipchat... not sure if you saw it so pasting here

it again has in its name "pdf"... but from the content it's not clear it's working with PDF
fr example the method above parseResponseContainsJson... in the content uses json_decode()... so it's working with JSON only
but that one that has PDF in the method name... could work even with "TEXT" files...
that's what I meant with the coment